packages/ui-kit/src/components/markdown.tsx

```diff
@@ -255,7 +256,8 @@ const InlineCode = styled.code`
   border: 1px solid ${colors.light.borderSecondary};
   border-radius: ${dimensions.spacings.xs};
   color: ${colors.light.textCode};
-  font-family: ${typography.fontFamilies.code};
+  font-family: ${typography.fontFamilies.nonMonospaceCode};
```

Comment:
is it ok to use a non monospace font for code? If I understand correctly it's only for inline code. 
Also, we still use `roboto-mono` for code snipplets right? Shouldn't we use the same monospace font for all code (being it inline or not)?

Reply:
I think the name "code" should be refactored in this case.  The use case are inline string literals, i.e. nothing where you have multiple indented lines or lists that need to be aligned.   So it can use a non-monospace font that has good letter disambiguation but does not have to be strictly monospace.